tests/unit/test_footprinter.cc

```diff
@@ -98,6 +113,15 @@ TEST_F(TestFootprinter, test_fundamental_types) {
     delete ptr;
   }
 
+  {
+    int i = 3;
```

Comment:
@cz4rs I just found that this generates a warning:

```
/Users/pbmille/repos/vt/lib/checkpoint/tests/unit/test_footprinter.cc:117:9: warning: declaration shadows a local variable [-Wshadow]
    int i = 3;
        ^
/Users/pbmille/repos/vt/lib/checkpoint/tests/unit/test_footprinter.cc:80:7: note: previous declaration is here
  int i;
      ^
```

Please open an issue and address it, by using a different name or something

Reply:
#134 created